### DISCUSSION TITLE
793

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#967] [Comments]
# 793

예전부터 하고 싶은 일만 하고 살 순 없을까 생각을 많이 했다.근데 그 내면을 살펴보면 사실은 "해야하지만 하기 싫은 이 일을 그만 둘 수는 없을까?" 에 가까웠다.아침에 일찍 일어나기 싫었고, 수업 빈 시간마다 아르바이트를 하러 다니는 것도 싫었다.근데 지금은 안다.그때 아침에 일찍 일어나는 습관을 만들어서 지금도 아침 시간을 활용해 하고 싶은 일을 할 수 있고,그때 시간이 빌 때마다 아르바이트를 해서 사회 생활을 시작할때 빚 없이 시작할 수 있었다.그리고 그런 시간들이 모여 결국 졸업 직전에 개발자로 방향을 전환해도 큰 부채감 없이 몰입할 수 있었다.아르바이트를 하며 차곡차곡 모아둔 돈으로 서울에서 별도의 아르바이트 없이 프로그래밍 공부에 집중할 수 있었고,아침마다 출근길을 피해 일찍가서 남들보다는 ..

https://jojoldu.tistory.com/793

**Comment by msbaek** [comment]
우와 멋지네요 ^^

**Reply by jojoldu** [reply]
ㅋㅋㅋㅋㅋ 감사합니다 명석님 ㅋㅋㅋㅋ 뭔가.... 명석님이 그렇게 얘기해주시니 부끄럽네요 ㅋㅋㅋ

**Reply by msbaek** [reply]
아니에요 동욱님 글을 보며 제가 가끔 느꼈던 아쉬움에 대해서 잘 정리할 수 있게되었습니다.
안다고 생각했는데 가끔 잊는 것들이 있네요.
앞으로도 동욱님의 멋진 활약 응원하겠습니다.

**Reply by jojoldu** [reply]
감사합니다 명석님.
명석님이 그렇게 얘기해주시니 저도 제 생각을 정리하길 잘했다는 생각이 드네요 ㅎㅎ
저도 명석님의 브런치 글 매번 잘 보고, 팀에도 잘 공유하고 있습니다 :)
항상 응원합니다 명석님 :)
